packages/appstore/src/components/cfds-listing/cfds-listing.scss

```diff
@@ -2234,3 +2236,57 @@
         }
     }
 }
+
+.cfd-password-change {
+    &-list {
+        list-style: disc;
+        margin-top: 0.4rem;
+        &-container {
+            padding-left: 2.5rem;
+            margin-bottom: 0.5rem;
+            font-size: 1.6rem;
+            color: var(--text-general);
+        }
+    }
+    &-modal {
+        &-description {
+            margin-bottom: 2rem;
+            @include mobile {
+                margin: 1rem 1rem 2rem;
+                &--button {
+                    display: flex;
+                    width: 100%;
+                }
+                &--footer {
+                    padding: 1rem 1.6rem;
+                    border-top: 1px solid #e9eced;
```

Comment:
@hamza-deriv Is this color solid for both light and dark theme?

Reply:
They didn't provide design for dark theme

Reply:
changed to `var(--general-hover)` as other dialogs